### PR TITLE
SDK-1573: Fixed composer timeout

### PR DESCRIPTION
**Pull request by vol4onok**
- Fixed timeout time.
https://spryker.atlassian.net/browse/SDK-1573

### CODE REVIEW

File: src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php [hunk 1]
```diff
@@ -153,6 +154,6 @@ protected function createResponse(Process $process): ResponseDto
         $output = $process->getExitCode() ? $process->getErrorOutput() : '';
         $outputs = array_filter([$command, $output]);
 
-        return new ResponseDto($process->isSuccessful(), implode(PHP_EOL, $outputs));
+        return new ResponseDto(!$process->getExitCode(), implode(PHP_EOL, $outputs));
```
Comment (dereuromark) [inline src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php:157]: Why this change?

**Comment by DmytroKlymanSpryker** [comment]
```
FILE: ...cture/PackageManager/CommandExecutor/ComposerCommandExecutor.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 13 | ERROR | [x] Type
    |       |     SprykerSdk\SdkContracts\Entity\ContextInterface is
    |       |     not used in this file.
    |       |     (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 46.66 secs; Memory: 34MB
```

File: src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php (hunk 1 above)
Comment (DmytroKlymanSpryker) [reply src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php:157]: @vol4onok , I also don't understand why have you changed it here. `isSuccessful` already checks the exit code: 

```
public function isSuccessful()
{
    return 0 === $this->getExitCode();
}
```

When we reach the timeout, exit code will be null (if I understand it correct) and isSuccessful() will return false.

Comment (dereuromark) [reply src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php:157]: Also more clear for humans :)

File: src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php [hunk 2]
```diff
@@ -19,12 +20,12 @@ class ComposerCommandExecutor implements ComposerCommandExecutorInterface
...
      * @var string
      */
-    protected const REQUIRE_COMMAND_NAME = 'composer require';
+    protected const REQUIRE_COMMAND_NAME = 'COMPOSER_PROCESS_TIMEOUT=0 composer require';
 
     /**
      * @var string
      */
-    protected const UPDATE_COMMAND_NAME = 'composer update';
+    protected const UPDATE_COMMAND_NAME = 'COMPOSER_PROCESS_TIMEOUT=0 composer update';
```
Comment (DmytroKlymanSpryker) [inline src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php:28]: I would personally prefer to create a new constant, e.g. `COMPOSER_TIMEOUT` or `COMPOSER_PROCESS_TIMEOUT`, define it as 600 or 900, and extend the places that call composer

File: src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php (hunk 1 above)
Comment (vol4onok) [reply src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php]: reverted

**Review by DmytroKlymanSpryker — APPROVED** [review]
